languages/ar.ini

```diff
@@ -71,7 +71,7 @@ and = "و"
 anonymous_tags = "وسوم مجهولة"
 APA Citation = "APA استشهاد"
 APA Edition Citation = "توثيق جمعية علم النفس الأمريكية APA (الطبعة السابعة)"
-applied_filter = "مرشح مطبّق"
+applied_filter = "مرشح مطبّق:"
```

Comment:
I'm not 100% certain this is correct, but it does seem to be consistent with other existing translation strings in the file that contain a colon.

As far as including a colon in a string that does not include tokens, I think it's appropriate for a value that is used as a label, because there are other structural aspects of the HTML that logically separate the parts (e.g. label and value are in separate markup areas, different table cells, etc.).

Reply:
Update: I've sent an email to our Arabic translators to get verification of this. If it's okay, I think we're ready to merge.

Reply:
Further update: I have received confirmation that this is correct. It's quite confusing, because in GitHub, the .ini file displays with the colon on the right side, which would be incorrect... but when the language files get interpreted by VuFind and displayed in right-to-left mode, the colon is correctly on the left where it belongs. See:

![image](https://user-images.githubusercontent.com/309069/202774671-deaca491-a68e-4ecd-b791-6a3655903c45.png)